### DISCUSSION TITLE
materail names not changeable

### DISCUSSION THREAD

**Discussion by YTTRR** [Zrips/CMI#6471] [Enhancement]
![图片](https://user-images.githubusercontent.com/70329799/145701690-31e98359-701b-4295-ad07-f669526d5291.png)
This is the rank module in CMI.
The materail names such as the red(&4) words in the picture are not able to be translated into other languages.
Could you add the translation option for the materail names in the future update? Thank you!

**Comment by Zrips** [comment]
One of todo thinks in the list. This will be added into CMILib plugin at some point in the future.
